### PR TITLE
Fix the enter pressed

### PR DESCRIPTION
**Pull request by maydemirx**
If developer set the sourceData option to custom ajax call function the
message will appear immediatly. It's not wait for ajax response

### CODE REVIEW

File: src/leaflet-search.js [hunk 1]
```diff
@@ -592,7 +592,8 @@ L.Control.Search = L.Control.extend({
 			case 13: //Enter
 				if(this._countertips == 1)
 					this._handleArrowSelect(1);
-				if (!this.options.sourceData || typeof this.options.sourceData !== 'function') {
+				if (!this.options.sourceData || typeof this.options.sourceData !== 'function'
+					|| $('.search-tip-select').text() == this._input.value) {
```
Comment (stefanocudini) [inline src/leaflet-search.js]: this Leaflet plugin don't require jQuery!

**Comment by stefanocudini** [comment]
@maydemirx the Leaflet plugins don't require jQuery!

**Comment by stefanocudini** [comment]
can you remove jQuery from core code.. and resend me a new pull request

**Comment by kaio328** [comment]
Sorry to reopen this discussion but pressing the enter button still results in closing the search field instead of handling the search...

**Pull request closed without merging**